matching-notification-api/src/main/java/org/phenotips/matchingnotification/finder/internal/DefaultMatchFinderManager.java

```diff
@@ -70,6 +72,22 @@ public void findMatchesForAllPatients(Set<String> serverIds, boolean onlyCheckPa
         }
     }
 
+    @Override
+    public Response findMatchesForPatient(Patient patient, String serverId)
+    {
+        this.logger.error("Finding matches for patient [{}] for server [{}]", patient.getId(), serverId);
+
+        for (MatchFinder matchFinder : this.matchFinderProvider.get()) {
+            Set<String> supportedServers = matchFinder.getSupportedServerIdList();
+            if (!supportedServers.contains(serverId)) {
```

Comment:
This part (checking if server is supported) is done inside `matchFinder.findMatches()` for the other `findMatches()` call, for consistency it makes more sense to do it there for this version as well. Also i would avoid overloading the method and would call it `findmatchesForASinglePatient()` or something along those lines

Reply:
Minor, code won't be that much cleaner if the check is moved inside.